.github/workflows/wc-check-textlint.yaml

```diff
@@ -24,13 +24,13 @@ jobs:
           # This is important to fetch the changes to the previous commit
           fetch-depth: 0
 
-      # https://github.com/oven-sh/setup-bun
-      - uses: oven-sh/setup-bun@4bc047ad259df6fc24a6c9b0f9a0cb08cf17fbe5 # v2
+      # https://github.com/pnpm/action-setup
+      - uses: pnpm/action-setup@fe02b34f77f8bc703788d5817da081398fad5dd2 # v4.0.0
+        name: Install pnpm
         with:
-          bun-version: latest
-      - run: bun install
+          version: 9
```

Comment:
![nits-badge](https://img.shields.io/badge/review-nits-orange.svg) 
CI で次のエラーが出ていました。
`package.json` の `packageManager` でバージョン指定するようにしているので、ここの指定は不要そうです！（ `.github/workflows/deploy-ticket.yaml` の修正もお願いします！ ）

```
Error: Multiple versions of pnpm specified:
    - version 9 in the GitHub Action config with the key "version"
    - version pnpm@9.12.3 in the package.json with the key "packageManager"
  Remove one of these versions to avoid version mismatch errors like ERR_PNPM_BAD_PM_VERSION
      at readTarget (/home/runner/work/_actions/pnpm/action-setup/fe02b34f77f8bc703788d5817da081398fad5dd2/dist/index.js:1:4528)
      at runSelfInstaller (/home/runner/work/_actions/pnpm/action-setup/fe02b34f77f8bc703788d5817da081398fad5dd2/dist/index.js:1:3742)
      at async install (/home/runner/work/_actions/pnpm/action-setup/fe02b34f77f8bc703788d5817da081398fad5dd2/dist/index.js:1:2976)
      at async main (/home/runner/work/_actions/pnpm/action-setup/fe02b34f77f8bc703788d5817da081398fad5dd2/dist/index.js:1:444)
  Error: Error: Multiple versions of pnpm specified:
    - version 9 in the GitHub Action config with the key "version"
    - version pnpm@9.[12](https://github.com/FlutterKaigi/2024/actions/runs/11645367023/job/32428323558?pr=451#step:3:13).3 in the package.json with the key "packageManager"
```

Reply:
ありがとうございます! 削除しました
8d9aed104705ff6fb08f663e1a15647f128ea19f